server/app/views/admin/AdminLayout.java

```diff
@@ -101,12 +122,35 @@ private ContainerTag renderNavBar() {
     String intermediaryLink = routes.TrustedIntermediaryManagementController.index().url();
     String apiKeysLink = controllers.admin.routes.AdminApiKeysController.index().url();
 
+    String activeNavStyle =
+        StyleUtils.joinStyles(
+            BaseStyles.TEXT_SEATTLE_BLUE,
+            Styles.FONT_MEDIUM,
+            Styles.BORDER_B_2,
+            BaseStyles.BORDER_SEATTLE_BLUE);
+
+    NavPage activePage = activeNavPage.orElseThrow();
+
     return adminHeader
-        .with(headerLink("Programs", programLink))
-        .with(headerLink("Questions", questionLink))
-        .with(headerLink("Versions", versionLink))
-        .with(headerLink("Intermediaries", intermediaryLink))
-        .with(headerLink("API keys", apiKeysLink))
+        .with(
+            headerLink(
+                "Programs", programLink, activePage == NavPage.PROGRAMS ? activeNavStyle : ""))
```

Comment:
This pattern is somewhat inelegant in that we have a text description and an equality check for every link. I could imagine trying to come up with something more `switch`-based, but the fact that there are only 5 sections led me to think the benefit wasn't really worth it given the small scope.

Reply:
Suggestion (non-blocking): an alternative could be to encapsulate this and the logic already in `headerLink` in a `AdminHeaderLink` class with a `render(NavPage navPage)` method

Reply:
Indeed. One wrinkle to doing that or other helpers around the nav page is that we also use "headerLink" for rendering the logout button, which isn't really nav per se.

If we wanted style reuse, we could either add LOGOUT as an enum value. This feels like breaking the encapsulation of what the enum represents.

Another alternative would be to extract out the styles common to both. At that point, I think we'd have more boilerplate around all of this and it wouldn't necessarily improve readability. 

Assuming that seems reasonable to you, I'm going to keep as-is for now.

Reply:
sgtm, thanks for talking it through with me!